app/views/users/_sidebar_additional.html.erb

```diff
@@ -7,7 +7,7 @@
           <h4><span class="emoji"><img src="<%= asset_path("github-logo.svg") %>" alt="github logo"></span><%= repo.name %></h4>
         </header>
         <div class="widget-body">
-          <%= repo.description %>
+          <%= repo.description.present? ? EmojiConverter.call(repo.description) : repo.description %>
```

Comment:
No need to render if the description is not present, something like this it's fine:

```suggestion
<% if repo.description %>
  <%= EmojiConverter.call(repo.description) %>
<% end %>